velox/exec/tests/utils/LocalExchangeSource.cpp

```diff
@@ -61,15 +67,34 @@ class LocalExchangeSource : public exec::ExchangeSource {
     VELOX_CHECK(requestPending_);
     auto requestedSequence = sequence_;
     auto self = shared_from_this();
-    buffers->getData(
-        taskId_,
-        destination_,
-        maxBytes,
-        sequence_,
-        // Since this lambda may outlive 'this', we need to capture a
-        // shared_ptr to the current object (self).
-        [self, requestedSequence, buffers, this](
+    auto hasBeenCalled = std::make_shared<bool>(false);
```

Comment:
Is this a bug fix? If so, would you put it in a separate PR and provide a description of what was wrong and how you are fixing it?